app/assets/stylesheets/era_custom.scss

```diff
@@ -16,3 +19,42 @@ div.terms-of-use-custom {
   margin-left: 30px;
   margin-right: 30px;
 }
+
+.info-pane p > a {
+  text-decoration: none;
+  color: $splash-content-link;
+}
+
+#featured_content img {
+  padding: 0px;
+  border: unset;
+}
+
```

Comment:
@murny  
Tried removing the ID-based CSS but this one is stubborn. Options as I see them:
* leave as ID-base to override pre-existing selector; aim easy merging future upstream changes by not changing avalon.scss or
* remove pre-existing ID-based selector in avalon.scss so there is no need to override https://github.com/ualbertalib/avalon/blob/feature/theme-mid-372/app/assets/stylesheets/avalon.scss#L462-L464